packages/xo-web/src/xo-app/menu/index.js

```diff
@@ -111,7 +111,20 @@ export default class Menu extends Component {
     () => this.state.proxyStates,
     proxyStates => some(proxyStates, state => state.endsWith('-upgrade-needed'))
   )
+  _getNProxiesErrors = createSelector(
+    () => this.state.proxyStates,
+    (proxyStates) => {
+      let count = 0
+
+      forEach(proxyStates, proxyState => {
+        if (proxyState === 'error') {
+          count++
+        }
+      })
 
+      return count
+    }
```

Comment:
With lodash's `countBy`:
```suggestion
    proxyStates => countBy(proxyStates).error
```